source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -241,6 +247,15 @@ export class Carousel extends Container {
         const containerForAdorners: HTMLElement = document.createElement("div");
         containerForAdorners.className = this.hostConfig.makeCssClassName("ac-carousel-container");
 
+        if (this.header) {
+            this.header.setParent(this);
+            let renderedHeader = this.header?.render();
```

Comment:
nitpick: We already checked for null above, so we should have to check again. Also, `renderedHeader` could be `const`.

```suggestion
            const renderedHeader = this.header.render();
```